core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -1887,7 +1891,7 @@ private[spark] class BlockManager(
    * but rather shadows them.
    * Requires an Indexed based shuffle resolver.
```

Comment:
I think that the comment needs to be updated to reflect what the return Boolean indicates.

Reply:
Good catch